plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -45,6 +69,17 @@ var JobListWidget = View.extend({
             obj[status.text] = true;
             return obj;
         }, {});
+        this.jobCheckedStates = {};
+        this.jobHighlightStates = {};
+        Object.defineProperty(this, 'jobs', {
```

Comment:
Unless there's an important reason for these attributes to be assigned this way, I'd strongly prefer that these be created using the normal Backbone syntax of `extend`. Also, these should be private (prefixed with a `_`).